ge/org.osate.ge.fx/src/org/osate/ge/fx/palette/PaletteGroup.java

```diff
@@ -0,0 +1,89 @@
+package org.osate.ge.fx.palette;
+
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.ToggleButton;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Font;
+
+/*
+ * Represents a single group in the palette.
+ *
+ */
+class PaletteGroup<G, I> extends Region {
+	final VBox buttonBox = new VBox();
+	final ToggleButton groupButton;
+
+	final String IDLE_GROUP_STYLE = "-fx-background-color: linear-gradient(rgb(247,247,247),rgb(200,200,200));"
```

Comment:
These constants should be private and static. Please place them at the top of the class.

Reply:
I have made the above changes.